core/kazoo_json/src/kz_json.erl

```diff
@@ -94,8 +94,7 @@
 -include_lib("kazoo/include/kz_log.hrl").
 -include_lib("kazoo_json/include/kazoo_json.hrl").
 
--export_type([json_term/0, api_json_term/0, json_terms/0
```

Comment:
Let me reformulate: you renamed a supertype. Why did you remove some subtypes though?